packages/explorer/src/app/(explorer)/[chainName]/worlds/[worldAddress]/observe/TimeAgoCell.tsx

```diff
@@ -0,0 +1,20 @@
+import { useEffect, useState } from "react";
+import { timeAgo } from "../../../../utils/timeAgo";
+
+export function TimeAgoCell({ timestamp }: { timestamp: bigint }) {
```

Comment:
this doesn't seem particularly cell-oriented so could prob name it:
```suggestion
export function TimeAgo({ timestamp }: { timestamp: bigint }) {
```
and I'd tend to leave out the color styling of this and let it inherit the color of the parent

Reply:
yeah makes sense, renamed to `TimeAgo`. As for styling, in order to do that it would require to be wrapped with a `div` element for that column just for this style. I understand the rationale to keep shared styling at parent-level as much as possible but in this case, each column has different styling when it comes to colors, and don't have a common parent in that regard.